src/libs/ReportUtils.js

```diff
@@ -806,9 +806,10 @@ function getIconsForParticipants(participants, personalDetails) {
  * @param {Object} report
  * @param {Object} personalDetails
  * @param {*} [defaultIcon]
+ * @param {Boolean} [isPayer]
```

Comment:
Fun fact: 70% time of the PR is for this param's name 😅

Reply:
Hahah unfortunately not many great options

isPaymentInitiator
isMoneySender

Reply:
Just a little clear-up because I'm messed up myself between `payer` and `payee`; `manager` and `owner` right now 😅:

| Action  | `owner` | `manager` | payer | payee |
|---------|---------|-----------|-------|-------|
| Request | A       | B         | B     | A     |
| Send    | B       | A         | A     | B     |

Payer is the one who pays; while payee is the one to whom money is paid. In which ever case, **the `manager` is the payer**.

In addition, I spotted another place where `isPayer` is used:

https://github.com/Expensify/App/blob/993b4ce4b5adb4fc17e050b2b10f40bb5f0544df/src/components/MoneyRequestHeader.js#L87-L88

Therefore, I think **`isPayer` is appropriate** here.